### PR TITLE
Added 4 icons

### PR DESCRIPTION
**Pull request by SagarDevAchar**
# Description

Added 4 icons

### Icons added
* edX (`org.edx.mobile`)
* OneStop (`com.swciitg.onestop2`)
* EveryCircuit (`com.everycircuit.free`)
* PO Pixel (`com.teenageengineering.pocketoperatorforpixel`)

## Contributor's checklist
- [x] I have followed the [Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md)
- [x] I have ensured that Lawnicons builds correctly
- [x] I am willing to make changes to my icons if someone suggests changes

### CODE REVIEW

**Review by x9136 — CHANGES_REQUESTED** [review]
Thanks for contributing! All icons have similar flaws: wrong size and use of fill. There is also an unacceptable width of the stroke.

Try to keep the size of all icons to 160×160px (either X×160px or 160×Xpx). The fill should be removed. As for stroke widths, the main options are 12px, 10px, and 8px. 6px is fine for details, and if the icon visually takes up little space, this is balanced by a 14px stroke. 

If you want to make changes, I suggest taking 10 minutes to read [the guides](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md#artboard--grid) before doing so. This will help make suitable icons and speed up the merge.

**Comment by SagarDevAchar** [comment]
Hey @x9136,

I have modified the edX icon as per the guidelines. Kindly look into it and let me know if I have complied to the lawnicons standards. 

Once you green flag this, I will perform the necessary changes to the other three icons as well and request a re-review.

**Comment by x9136** [comment]
@SagarDevAchar ✌️

Everything became according to the guides.

If you want, you can improve the quality. For example, if you use 10px stroke, you can make the letters bigger and the "e" will look better. I would also raise the icon a few pixels higher, because visually it's pressed to the bottom because of the jagged top edge.

**Comment by SagarDevAchar** [comment]
@x9136 

By the way, I needed some inputs on the `everycircuit` icon design. The icon has complex features and my interpretation has been as shown below. Stroke width is `10px`.

everycircuit

**Comment by x9136** [comment]
@SagarDevAchar Visually the icon is ok. When reducing the size, you'll have to use a stroke thickness of 8px in the center part so that the icon doesn't turn into a mess. I got it like this:

Screenshot 2023-10-14 at 16 59 48

File: svgs/po_pixel.svg
Comment (x9136) [inline svgs/po_pixel.svg:1]: Screenshot 2023-10-14 at 17 34 22
It's worth moving the icon 1px higher.

**Review by x9136 — CHANGES_REQUESTED** [review]

File: svgs/po_pixel.svg
Comment (SagarDevAchar) [reply svgs/po_pixel.svg:1]: Whoops, looks like an accidental keypress!

**Pull request merged**